instrumentation/servlet-user/src/main/java/javax/servlet/Servlet.java

```diff
@@ -26,7 +26,7 @@ public void service(ServletRequest request, ServletResponse response) {
         if (request instanceof HttpServletRequest) {
             Principal principal = ((HttpServletRequest) request).getUserPrincipal();
             if (principal != null) {
-                AgentBridge.getAgent().getTransaction().getAgentAttributes().put("user", principal.getName());
+                NewRelic.setUserName(principal.getName());
```

Comment:
This does the same thing but adds additional useful logs and a supportability metric, `SetUserName`

Reply:
didn't this use a different parameter name?